docs/agent_container_manual_deploy.md

```diff
@@ -0,0 +1,76 @@
+## Instructions for starting an agent in a container on Linux
+These instructions assume that the agent container you wish to deploy already exists and is in a container image repository somewhere.
+
+### Prerequisites
+Docker needs to be installed on the host device. For instructions on installing Docker on Linux see [here](https://docs.docker.com/engine/install/).
+
+Need to have the SSL certificate from the management hub.
+
+### Starting the agent
+Before starting the container, create a config file for the agent. This should include the following:
+ * HZN_EXCHANGE_URL=<adress of your exchange>
+ * HZN_FSS_CSSURL=<css address>
+ * HZN_ORG_ID=<the org this node should be in>
+ * HZN_EXCHANGE_USER_AUTH=<exchange username:password>
+Optional:
+ * HZN_DEVICE_ID=<a name for your node>
+ * HZN_MGMT_HUB_CERT_PATH=<path to the ssl cert file one the host machine>
+       * Note: if this is set, then the cert file must be mounted to the same filepath in the container.
+
+
+
+Prior to starting the container, the following variables need to be set.
+ * HZN_AGENT_IMAGE=<name of the agent container image in the repo>
+ 	* This should include the url for the repository if it's not the default one.
+ * HZN_AGENT_IMAGE_TAG=<version of the agent container to use>
+ * CONFIG_FILE=<Location of the config file>
+ 	* The config file with the contents from above.
+ * HZN_MGMT_HUB_CERT_MOUNT="-v <ssl cert file on host>:<$HZN_MGMT_HUB_CERT_PATH>"
+ 	* This needs to be set in the config file so the agent can find the cert after it starts.
+ * DOCKER_NAME=<name for the agent container>
+ * HORIZON_AGENT_PORT=<port number>
+ 	* The port to expose from the container that hzn will call the agent on. Typically 8081.
+ * DOCKER_ADD_HOSTS="--add-host=$<host name to add to container hosts file>"
+ 	* Omit if not needed
+
+Example:
+```bash
+HZN_AGENT_IMAGE=openhorizon/amd64_anax
+HZN_AGENT_IMAGE_TAG=latest
+CONFIG_FILE=/etc/default/horizon
+HZN_MGMT_HUB_CERT_PATH=/etc/default/horizon/agent.crt
+DOCKER_NAME=horizon1
+HORIZON_AGENT_PORT=8081
+```
+
+```bash
+docker run $DOCKER_ADD_HOSTS -d --restart always --name $DOCKER_NAME --privileged -p 127.0.0.1:$HORIZON_AGENT_PORT:8510 -e DOCKER_NAME=$DOCKER_NAME -v /var/run/docker.sock:/var/run/docker.sock -v $CONFIG_FILE:/etc/default/horizon:ro $HZN_MGMT_HUB_CERT_MOUNT -v $DOCKER_NAME_var:/var/horizon/ -v $DOCKER_NAME_etc:/etc/horizon/ -v $DOCKER_NAME_fss:/var/tmp/horizon/$DOCKER_NAME $HZN_AGENT_IMAGE:$HZN_AGENT_IMAGE_TAG
+
+export HORIZON_URL=http://localhost:$HORIZON_AGENT_PORT
```

Comment:
explain in a note that this export enables the CLI to talk to the agent in the container.

Reply:
Done